### PR TITLE
add a dockerfile for forklift-api

### PR DESCRIPTION
**Pull request by ShakedAviv50313**

### CODE REVIEW

**Review by fabiand — CHANGES_REQUESTED** [review]
Please use the name `Containerfile`

File: build/forklift-api/Containerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Required to be able to get files from within the pod
+RUN microdnf -y install tar && microdnf clean all
+
+COPY --from=builder /app/forklift-api /usr/local/bin/forklift-api
+ENTRYPOINT ["/usr/local/bin/forklift-api"]
+LABEL \
+        com.redhat.component="forklift-api-container" \
+        name="forklift/forklift-api-rhel9" \
+        io.k8s.display-name="Forklift" \
+        io.k8s.description="Forklift - API" \
```
Comment (yaacov) [inline build/forklift-api/Containerfile:18]:
```suggestion
        io.k8s.description="Forklift API server is running Kubernetes validation and mutation webhooks for Forklift resources." \
```

**Review by yaacov — APPROVED** [review]
LGTM

**Comment by yaacov** [comment]
@ShakedAviv50313 hi, your commit incorrectly signed off, it should have a "<" and ">" around the email, can you re-sign your commit correctly ?

File: build/forklift-api/Containerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ENTRYPOINT ["/usr/local/bin/forklift-api"]
+LABEL \
+        com.redhat.component="forklift-api-container" \
+        name="forklift/forklift-api-rhel9" \
+        io.k8s.display-name="Forklift" \
+        io.k8s.description="Forklift API server is running Kubernetes validation and mutation webhooks for Forklift resources." \
+        io.openshift.tags="migration,forklift,mtv" \
+        summary="Forklift- API" \
+        description="Forklift - API" \
+        maintainer="Migration Toolkit for Virtualization Team <migtoolkit-virt@redhat.com>"
```
Comment (ernesgonzalez33) [inline build/forklift-api/Containerfile:22]: @ShakedAviv50313 please change for `"Forklift by Konveyor Community <forklift-dev@googlegroups.com>"`

**Review by mnecas — APPROVED** [review]
lgtm

**Pull request merged**